spec/lib/local-links-manager/import/local_custodian_codes_importer_spec.rb

```diff
@@ -0,0 +1,42 @@
+describe LocalLinksManager::Import::LocalCustodianCodesImporter do
+  describe "#import_from_csv" do
+    context "local custodian codes for Local Authorities in Great Britain" do
+      let!(:local_authority_1) { create(:local_authority, slug: "leicestershire", local_custodian_code: nil) }
+      let!(:local_authority_2) { create(:local_authority, slug: "surrey-heath", local_custodian_code: nil) }
+      let!(:local_authority_3) { create(:local_authority, slug: "stockton-on-tees", local_custodian_code: nil) }
+
+      it "populates code with value in different formats" do
+        LocalLinksManager::Import::LocalCustodianCodesImporter.new.import_from_csv("spec/lib/local-links-manager/import/fixtures/addressbase-local-custodian-codes-test.csv")
```

Comment:
This can be refactored slightly:

```suggestion
        described_class.new.import_from_csv("spec/lib/local-links-manager/import/fixtures/addressbase-local-custodian-codes-test.csv")
```